src/main/java/com/minecrafttas/common/server/Client.java

```diff
@@ -78,6 +81,12 @@ public Client(String host, int port, PacketID[] packetIDs, String name) throws E
 	 */
 	public Client(AsynchronousSocketChannel socket, PacketID[] packetIDs) {
 		this.socket = socket;
+		try {
+			this.socket.setOption(StandardSocketOptions.SO_KEEPALIVE, true);
+			this.socket.setOption(StandardSocketOptions.TCP_NODELAY, true);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
```

Comment:
just add throws and do proper error handling?

Reply:
Sure